tests/Unit/DataStructures/Test_FixedHashMap.cpp

```diff
@@ -303,7 +303,7 @@ struct RepeatedHash {
 };
 
 template <typename KeyType>
-// NOLINTNEXTLINE(google-readability-function-size)
+// NOLINTNEXTLINE(google-readability-function-size, readability-function-size)
```

Comment:
Did you check if clang-tidy is still throwing the old warning? I've seen cases where our newer clang-tidy (llvm 10) changes the warning for a given code line, so the cleaner fix is to replace (not add) the warning.

Reply:
My intention was to keep the nolint compatible with older versions. But I can also just upgrade the nolint.